guides/20_Builders_Journey.md

```diff
@@ -0,0 +1,476 @@
+
+### 2.1 Amazon Chime SDK for JavaScript
+
+The Amazon Chime SDK for JavaScript has everything you need to build custom calling and collaboration experiences in your web application, including methods to: configure meeting sessions, list and select audio and video devices, start and stop screen share and screen share viewing, receive callbacks when media events occur such as volume changes, and control meeting features such as audio mute and video tile bindings.
+
+The Amazon Chime SDK for JavaScript works by connecting to meeting session resources that you have created in your AWS account. Built on top of WebRTC, the Amazon Chime SDK has everything you need to build custom calling and collaboration experiences in your web application, including methods to: configure meeting sessions, list and select audio and video devices, start and stop screen share and screen share viewing, receive callbacks when media events occur such as volume changes, and control meeting features such as audio mute and video tile bindings. For additional information, please see:
+
+- Tutorial - [Workshop Studio – Building with Amazon Chime SDK](https://catalog.us-east-1.prod.workshops.aws/v2/workshops/de1c7d2d-6826-44db-a678-ff7a62747ec4/en-US/)
+- [WebRTC Tutorial](https://www.tutorialspoint.com/webrtc/index.htm)
+- SDK Tech Talk:[https://www.youtube.com/watch?v=v9KFalEv6gw](https://www.youtube.com/watch?v=v9KFalEv6gw)
+- SDK Video:[https://youtu.be/lNQ1_WfUQl4](https://youtu.be/lNQ1_WfUQl4)
+- [Amazon Chime SDK for JavaScript Overview](https://aws.github.io/amazon-chime-sdk-js/index.html)
+- [Getting Started](https://aws.github.io/amazon-chime-sdk-js/modules/apioverview.html#1-create-a-session)
+- Frequently Asked Questions
+  - [Amazon Chime SDK FAQ](https://aws.amazon.com/chime/chime-sdk/faq/) (Web Page)
+  - [Amazon Chime SDK FAQ](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html) (GitHub)
+  - [SA Internal Amazon Chime SDK FAQ](https://quip-amazon.com/2eChAFiHKcpi/Amazon-Chime-SDK-FAQ) (Internal Quip Document)
+- [Amazon Chime SDK Guides](https://github.com/aws/amazon-chime-sdk-js/tree/master/guides)(Please review each document in this list)
+- [A Builder's Guide to Amazon Chime sessions at](https://aws.amazon.com/blogs/business-productivity/a-builders-guide-to-amazon-chime-sessions-at-reinvent-2019/)[re:Invent 2019](https://aws.amazon.com/blogs/business-productivity/a-builders-guide-to-amazon-chime-sessions-at-reinvent-2019/)
+- [Serverless Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless)
+- [Developer Guide](https://docs.aws.amazon.com/chime/latest/dg/using-the-messaging-sdk.html)
+
+### 2.2 Quotas / Limits
+
+AWS has service quotas (previously call service limits) to help prevent accidental provisioning of more resources than needed and to help ensure availability of AWS resources.
+
+For each AWS account there are hard limits that cannot be adjusted and soft limits which can be adjusted:
+
+**Hard limits:**
```

Comment:
This is good to have here, but, we may have to track to update this when the limits are changed in AWS docs. So could you please add a link to AWS docs here.